shared/enclave/server/wallets/trezor.ts

```diff
@@ -25,6 +25,15 @@ async function getSession() {
       currentSession = null;
     }
   });
+  device.on('pin', (_, cb: (err: Error | undefined, pin?: string) => void) => {
```

Comment:
Might as well keep it consistent and have `err` parameter use the optional type syntax as used in `pin`

Reply:
My thought was that I didn't want you ever calling the callback with neither error nor pin, so I figured making err explicitly undefined would be good for that. But it's definitely more succinct to go optional, so I'll do that.